index.html

```diff
@@ -0,0 +1,1117 @@
+            </path>
+          </g>
+        </svg>
+      </div>
+      <div class="col-sm-6 pip col-md-8 col-lg-4 d-flex justify-content-end align-items-center pb-lg-1">
+        <button class="btn pr-lg-1 sign-in">SIGNIN</button>
+        <button class="btn btn-primary rounded-0 border subscribe-btn"><span class="pl-0">SUBSCRIBE</span></button>
+        <div class="d-md-block d-lg-none">
+          <i class="material-icons">menu</i>
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-10 disappear">
+      <ul class="col-lg-12 mr-2 d-flex justify-content-between list-bk-color mt-1">
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">U.S.</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">World</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Business</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Tech & Science</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Culture</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Newsgeek</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Sports</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Health</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">The Debate</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Vantage</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link">Weather</a>
+        </li>
+      </ul>
+    </div>
+    <input type="text" placeholder="Search" class="col-lg-1 search-input mt-lg-2">
```

Comment:
Use semantic HTML. In this case, you can change the div that contains your list and change it for a `<nav>` tag

Please check the size and spacing of your `<li>` elements. The original page has a different look (font-size, less padding...)
Yours:
![Screenshot from 2020-04-13 08-01-22](https://user-images.githubusercontent.com/51926937/79122523-7bc28e80-7d5d-11ea-88ca-065a83d9670e.png)

Original:
![Screenshot from 2020-04-13 08-05-12](https://user-images.githubusercontent.com/51926937/79122540-85e48d00-7d5d-11ea-8eab-5abcc629d052.png)